pkgs/development/python-modules/tinygrad/default.nix

```diff
@@ -209,7 +213,10 @@ buildPythonPackage rec {
     changelog = "https://github.com/tinygrad/tinygrad/releases/tag/v${version}";
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [ GaetanLepage ];
-    # Requires unpackaged pyobjc-framework-libdispatch and pyobjc-framework-metal
-    broken = stdenv.hostPlatform.isDarwin;
+    badPlatforms = [
+      # Requires unpackaged pyobjc-framework-libdispatch and pyobjc-framework-metal
+      "aarch64-darwin"
+      "x86_64-darwin"
```

Comment:
This should be `lib.systems.inspect.predicates.isDarwin`

Reply:
> This should be `lib.systems.inspect.predicates.isDarwin`

You mean `[ lib.systems.inspect.patterns.isDarwin ]` ?